packages/dapp-svelte-wallet/react-ui/src/components/WalletConnection.js

```diff
@@ -94,11 +94,17 @@ const WalletConnection = ({ setConnectionState, connectionState }) => {
   }, []);
 
   return (
-    <div className={classes.connector}>
-      <Typography variant="body1">
-        Connection Status:{' '}
-        {connectionState === 'admin' ? 'Connected' : 'Disconnected'}
-      </Typography>
+    <div className={`connector ${classes.connector}`}>
```

Comment:
You might want to look at `clsx` (spelling?), for a clearer way of adding to the class list.

Reply:
> I think in my next PR I may try using the web components from material directly (https://github.com/material-components/material-web), since this library has been a bit frustrating to work with.

That's a good thing to explore, but please don't rewrite the material-ui stuff you have already until the experiment is successful and you have spare time.